integration_tests/README.md

```diff
@@ -0,0 +1,56 @@
+# Integration Tests
+
+## Running Tests
+
+To run integration tests:
+
+`make integration`
+
+## Making Tests
+
+### 1. Fixtures
+
+Fixtures Directory: `integration_tests/fixtures`
+
+- Folders in the fixtures directory will be built as docker images in alphabetical order
+- Each folder should contain a `Dockerfile` and any resources necessary to build the image
+- The resulting image will be named `cbfix_fixture_name`
+
+*Note*: Since fixtures are created in alpha order, they can have FROM directives for previously created images
+
+### 2. Tests
+
+Tests Directory: `integration_tests/tests`
+
+- Folders in the tests directory will be run as tests
```

Comment:
I think it's clear to me and you but we should include a note here that these tests are run in the context of the testing host (i.e. TravisCI or your local environment), not inside a container.